classes/xt_quirks.bbclass

```diff
@@ -100,3 +100,6 @@ python do_xt_config_reconstruct() {
     with open(conf_file, "a") as f:
         f.write("require" + " " + "build-versions.inc" + "\n")
 }
+
+do_configure[nostamp] = "1"
```

Comment:
Will do_configure remove the lower Yocto and start the build from the start (with state caches used?)

Reply:
I have tested and in this case do_configure does not cause lower yocto removal.

Reply:
looks good then